src/emulator/downloadableEmulators.ts

```diff
@@ -51,16 +51,18 @@ const DownloadDetails: { [s in DownloadableEmulators]: EmulatorDownloadDetails }
     },
   },
   gui: {
-    version: "0.0.1",
-    downloadPath: path.join(CACHE_DIR, "gui-v0.0.1.zip"),
-    unzipDir: path.join(CACHE_DIR, "gui-v0.0.1"),
-    binaryPath: path.join(CACHE_DIR, "gui-v0.0.1", `server.bundle.js`),
+    version: "0.0.0",
+    downloadPath: path.join(CACHE_DIR, "gui-vEAP.zip"),
+    unzipDir: path.join(CACHE_DIR, "gui-vEAP"),
+    binaryPath: path.join(CACHE_DIR, "gui-vEAP", `server.bundle.js`),
     opts: {
       cacheDir: CACHE_DIR,
-      remoteUrl: "https://storage.googleapis.com/firebase-preview-drop/emulator/gui-v0.0.1.zip",
-      expectedSize: 1204964,
-      expectedChecksum: "52847b962bb66de639487d96a07a69d3",
+      remoteUrl: "https://storage.googleapis.com/firebase-preview-drop/emulator/gui-vEAP.zip",
```

Comment:
I think to be  compatible with semver tools (which we may not even be using, idk) this would be better as `v0.0.0-EAP`

Reply:
kk